javascript/account.js

```diff
@@ -0,0 +1,12 @@
+// 테스트 계정 설정
+const accountInfo = [
+  { email: 'test@codeit.com', password: 'codeit101' }, 
+]
+
+// 이메일 형식 확인하는 정규식
+const emailForm = /^[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*\.[a-zA-Z]{2,3}$/i
+
+// 비밀번호 형식 확인하는 정규식
+const passwordFrom = /^(?=.*[a-zA-Z0-9]).{8,15}$/
```

Comment:
이름을 봤을때는 정규식인지 모르겠어서 좀 더 명시적으로 네이밍을 하시면 좋을 것 같습니다.

```suggestion
// 이메일 형식 확인하는 정규식
const emailRegex = /^[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*\.[a-zA-Z]{2,3}$/i

// 비밀번호 형식 확인하는 정규식
const passwordRegex = /^(?=.*[a-zA-Z0-9]).{8,15}$/
```

Reply:
넵!! 네이밍을 좀 더 신경쓰도록 하겠습니다.